test/library/packages/Sort/RadixSort/radixsortMSB.execopts

```diff
@@ -0,0 +1 @@
+--printStats=false
```

Comment:
Let's use the large problem size as other [sort performance tests](https://chapel-lang.org/perf/chapcs/?graphs=linearithmicsortson224bytesofshuffleddata,quadraticsortson212bytesofshuffleddata) of 2^24 bytes of data, or an array of `2097152` integers, so that our timings are comparable to other sorts in the Sort module.

However, we don't want the correctness test to use this problem size, because we generally want to keep correctness tests quick. To handle this, we can introduce a `radixsortMSB.perfexecopts` with the following content:

```
--printStats=false --size=2097152
```

This will be the execution options used when testing with `start_test --performance`.